Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -27,7 +27,7 @@ public function fix(\SplFileInfo $file, $content)
 
         $content = rtrim($content);
 
-        if (strlen($content)) {
+        if ('' !== $content) {
```

Comment:
It's not so simple - there are several different patterns + yoda/no-yoda style doubles variants.
I can share all patterns from my analyser if you'll create a ticket for this.